### PR TITLE
Autokeras added in automl install script

### PR DESCRIPTION
**Pull request by Sette**

### CODE REVIEW

**Review by Wei-1 — COMMENTED** [review]
The current build is bugged by python2
we might need to fix it first

**Comment by Sette** [comment]
Can I change python version on travis?

**Comment by Wei-1** [comment]
I believe we change the travis setting by modifying the `.travis.yml` file.
https://github.com/apache/incubator-marvin/blob/develop/.travis.yml

File: .travis.yml [hunk 1]
```diff
@@ -38,8 +38,7 @@ matrix:
     - language: python
       os: linux
       python:
-        - 2.7
-        - 3.6
+        - 3.7
```
Comment (Wei-1) [inline .travis.yml:41]: Changing python version will have a series of other versioning issue.
We might have to open another PR for this first.

**Pull request closed without merging**